src/infrastructure/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,37 @@
+import ReactModal from 'react-modal';
+import styles from './Modal.module.scss';
+
+ReactModal.setAppElement('#root');
+
+export const Modal = ({ children, isOpen, handleClose }: ModalProviderProps) => (
+  <ReactModal
+    style={{
+      overlay: {
+        backgroundColor: '#00000090',
+      },
+      content: {
+        width: '30rem',
+        height: '40rem',
+        top: '50%',
+        left: '50%',
+        right: 'auto',
+        bottom: 'auto',
+        marginRight: '-50%',
+        transform: 'translate(-50%, -50%)',
+      },
+    }}
```

Comment:
There isnt a way to put this style in a scss file ?

Reply:
Sure, I will do this right now. Thanks